tartiflette/parser/visitor.py

```diff
@@ -66,16 +68,17 @@ def __init__(self, variables=None, schema_definition=None):
         self._current_argument_name = None
         self._current_type_condition = None
         self._current_fragment_definition = None
+        self._parent_node_type = None
         self._fragments = {}
-        self._schema_definition = schema_definition
+        self._schema_definition: GraphQLSchema = schema_definition
 
-    def _on_argument_in(self, element):
+    def _on_argument_in(self, element: _VisitorElement):
         self._current_argument_name = element.name
 
     def _on_argument_out(self, _):
         self._current_argument_name = None
 
-    def _on_value_in(self, element):
+    def _on_value_in(self, element: _VisitorElement):
```

Comment:
Nice, I want to cythonify all this at some point :).

Reply:
That would be amazing :D